libcaf_core/caf/blocking_response_handle.hpp

```diff
@@ -66,6 +67,48 @@ template <class Result>
 using blocking_delayed_response_handle_t =
   typename blocking_delayed_response_handle_oracle<Result>::type;
 
+template <class... Ts>
+struct expected_builder;
```

Comment:
Probably forgot to delete this? Probably added in an earlier commit, so please remove + squash.